sdk/keyvault/azure_security_keyvault_keys/src/lib.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+//
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// Code generated by Microsoft (R) Rust Code Generator. DO NOT EDIT.
+
+mod generated;
+
+pub use crate::generated::clients::*;
```

Comment:
This should be removed until @jhendrixMSFT fixes the emitted `generated/mod.rs`.